RSDKUtils.podspec

```diff
@@ -5,30 +5,50 @@ Pod::Spec.new do |s|
   s.summary      = "Rakuten's SDK Team internal utilities module."
   s.homepage     = "https://github.com/rakutentech/ios-sdkutils"
   s.license      = { :type => 'MIT', :file => 'LICENSE' }
-  s.source       = { :git => "https://github.com/rakutentech/ios-sdkutils", :tag => s.version.to_s }
-  s.platform     = :ios, '10.0'
+  s.source       = { :git => "https://github.com/rakutentech/ios-sdkutils.git", :tag => s.version.to_s }
+  s.platform     = :ios, '11.0'
   s.swift_version = '5.0'
   s.requires_arc = true
   s.pod_target_xcconfig = {
     'CLANG_ENABLE_MODULES'                                  => 'YES',
     'CLANG_MODULES_AUTOLINK'                                => 'YES',
     'CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES' => 'YES',
     'GCC_C_LANGUAGE_STANDARD'                               => 'gnu11',
-    'OTHER_CFLAGS'                                          => "'-DRPT_SDK_VERSION=#{s.version.to_s}'",
-    'FRAMEWORK_SEARCH_PATHS'                                => '"${PODS_CONFIGURATION_BUILD_DIR}/Nimble"'
+    'OTHER_CFLAGS'                                          => "'-DRPT_SDK_VERSION=#{s.version.to_s}'"
   }
   s.user_target_xcconfig = {
     'CLANG_ENABLE_MODULES'                                  => 'YES',
     'CLANG_MODULES_AUTOLINK'                                => 'YES',
     'CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES' => 'YES',
   }
   s.weak_frameworks = [
-    'Foundation',
-    'XCTest',
-    'Nimble'
+    'Foundation'
   ]
-  s.source_files = "Sources/RSDKUtils/**/*.{swift,m,h}"
-  s.public_header_files = "Sources/RSDKUtils/*.h,Sources/RSDKUtils/StandardHeaders/*.h,Sources/RSDKUtils/KeyStore/*.h"
-  s.module_map = 'Sources/RSDKUtils/RSDKUtils.modulemap'
+  s.module_map = './RSDKUtils.modulemap'
+  s.default_subspec = 'Main'
+
+  s.subspec 'Main' do |ss|
+    ss.source_files = "Sources/RSDKUtilsMain/**/*.{swift,m,h}"
+    ss.public_header_files = "Sources/RSDKUtilsMain/*.h"
+    ss.dependency 'RSDKUtils/RLogger'
+  end
+
+  s.subspec 'Tests' do |ss|
```

Comment:
Is it normal to have a Tests subspec?

Reply:
I'm not sure. I changed it from 'XCTest' but maybe 'TestUtils' is better?

Reply:
yeah, `TestUtils` or `TestHelpers` (so as not to repeat utils, though that doesn't really matter) seems better to me